backend/database/wrapper/tag_wrapper.py

```diff
@@ -0,0 +1,110 @@
+from typing import List
+
+from flask import current_app
+from sqlalchemy import func
+
+from backend.blueprints.spa_api.errors.errors import ReplayNotFound
+from backend.database.objects import Tag, Game, GameTag
+
+
+class TagWrapper:
+    @staticmethod
+    def create_tag(session, user_id: str, name: str) -> Tag:
+        tag = Tag(name=name, owner=user_id)
+        session.add(tag)
+        session.commit()
+        return tag
+
+    @staticmethod
+    def rename_tag(session, user_id: str, old_name: str, new_name: str) -> Tag:
+        tag = TagWrapper.get_tag(user_id, old_name, session)
+        tag.name = new_name
+        session.commit()
+        return tag
+
+    @staticmethod
+    def get_tags(session, user_id: str) -> List[Tag]:
```

Comment:
this is not consistent with the method below.

Also the with_session decoartor?